### ISSUE TITLE
cc 2.9.1 build with FFMPEG 4 fail

### ISSUE THREAD

**Issue by tigerfoot** [CloudCompare/CloudCompare#768]
## Current Behavior
Trying to build 2.9.1 with ffmpeg 4 release failed with errors

## Expected Behavior
Make CC finish without error even with ffmpeg 4

## Steps to Reproduce (for bugs)

[cc_ffmpeg4_fail_build.log](https://github.com/CloudCompare/CloudCompare/files/2195784/cc_ffmpeg4_fail_build.log)

Spec file is visible here
[Cloudcompare spec on obs](https://build.opensuse.org/package/show/home:bruno_friedmann:branches:Application:Geo/cloudcompare)

## Context/Comments
Seems the ffmpeg part need to be adjusted to newer ffmpeg 4 (but then how to keep it compatible with previous version (at least 3.0) for distribution that doesn't have ffmpeg 4

## Your Environment
* CC Version: 2.9.1 + patches.
* OS & Version:  openSUSE Tumbleweed 20180712 
Linux 4.17.4-1-default x86_64 GNU/Linux, nvidia: 390.67
Qt: 5.11.1, KDE Frameworks: 5.47.0, Plasma: 5.13.2, kmail2 5.8.2
gcc version 8.1.1 20180614
* Graphics card: nvidia Quadro M2000M

I found it would be cool to have it fixed before 2.10 release.

Summary of the errors
```log
[  443s] Dependee "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/DependInfo.cmake" is newer than depender "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/depend.internal".
[  443s] Dependee "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/depend.internal".
[  443s] Scanning dependencies of target QTFFMPEG_LIB_autogen
[  443s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] /usr/bin/make -f plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/build.make plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/build
[  443s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] [ 28%] Automatic MOC for target QTFFMPEG_LIB
[  443s] cd /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper && /usr/bin/cmake -E cmake_autogen /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/AutogenInfo.cmake RelWithDebInfo
[  443s] AutoMoc: Generating "moc_predefs.h" because it doesn't exist
[  443s] AutoMoc: Running command:
[  443s] /usr/bin/c++ -dM -E -c /usr/share/cmake/Modules/CMakeCXXCompilerABI.cpp -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/QTFFMPEG_LIB_autogen/include -I/usr/include/qt5 -I/usr/include/qt5/QtOpenGL -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I/usr/lib64/qt5/mkspecs/linux-g++ -I/usr/include/qt5/QtConcurrent -I/usr/include/qt5/QtPrintSupport -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/contrib/dxflib-3.17.0/src -I/usr/include/gdal -I/usr/include/ffmpeg -I/usr/include -DQT_USE_QSTRINGBUILDER -D_CRT_SECURE_NO_WARNINGS -D__STDC_CONSTANT_MACROS
[  443s] AutoMoc: Generating MOC compilation /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/QTFFMPEG_LIB_autogen/mocs_compilation.cpp
[  443s] AutoGen: Writing settings file "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB_autogen.dir/AutogenOldSettings.txt"
[  443s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] [ 28%] Built target QTFFMPEG_LIB_autogen
[  443s] /usr/bin/make -f plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/build.make plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/depend
[  443s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] cd /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1 /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/DependInfo.cmake
[  443s] Dependee "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/DependInfo.cmake" is newer than depender "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/depend.internal".
[  443s] Dependee "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/depend.internal".
[  443s] Scanning dependencies of target QTFFMPEG_LIB
[  443s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] /usr/bin/make -f plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/build.make plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/build
[  443s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  443s] [ 28%] Building CXX object plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o
[  443s] cd /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper && /usr/bin/c++  -DQT_USE_QSTRINGBUILDER -D_CRT_SECURE_NO_WARNINGS -D__STDC_CONSTANT_MACROS -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build/plugins/qAnimation/QTFFmpegWrapper/QTFFMPEG_LIB_autogen/include -I/usr/include/qt5 -I/usr/include/qt5/QtOpenGL -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I/usr/lib64/qt5/mkspecs/linux-g++ -I/usr/include/qt5/QtConcurrent -I/usr/include/qt5/QtPrintSupport -I/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/contrib/dxflib-3.17.0/src -I/usr/include/gdal -I/usr/include/ffmpeg  -fmessage-length=0 -grecord-gcc-switches -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fPIE -pie -fno-strict-aliasing -DNDEBUG -std=c++11 -fPIC -fopenmp -O2 -g -DNDEBUG   -o CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o -c /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp: In member function 'bool QVideoEncoder::open(QString*)':
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:120:18: warning: 'void av_register_all()' is deprecated [-Wdeprecated-declarations]
[  444s]   av_register_all();
[  444s]                   ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:2025:6: note: declared here
[  444s]  void av_register_all(void);
[  444s]       ^~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:120:18: warning: 'void av_register_all()' is deprecated [-Wdeprecated-declarations]
[  444s]   av_register_all();
[  444s]                   ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:2025:6: note: declared here
[  444s]  void av_register_all(void);
[  444s]       ^~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:182:32: error: 'CODEC_FLAG_GLOBAL_HEADER' was not declared in this scope
[  444s]    m_ff->codecContext->flags |= CODEC_FLAG_GLOBAL_HEADER;
[  444s]                                 ^~~~~~~~~~~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:182:32: note: suggested alternative: 'AV_CODEC_FLAG_GLOBAL_HEADER'
[  444s]    m_ff->codecContext->flags |= CODEC_FLAG_GLOBAL_HEADER;
[  444s]                                 ^~~~~~~~~~~~~~~~~~~~~~~~
[  444s]                                 AV_CODEC_FLAG_GLOBAL_HEADER
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:194:21: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   m_ff->videoStream->codec = m_ff->codecContext;
[  444s]                      ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:194:21: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   m_ff->videoStream->codec = m_ff->codecContext;
[  444s]                      ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:194:21: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   m_ff->videoStream->codec = m_ff->codecContext;
[  444s]                      ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp: In member function 'virtual bool QVideoEncoder::close()':
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:273:75: warning: 'int avcodec_encode_video2(AVCodecContext*, AVPacket*, const AVFrame*, int*)' is deprecated [-Wdeprecated-declarations]
[  444s]    int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, 0, &got_packet);
[  444s]                                                                            ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:6:
[  444s] /usr/include/ffmpeg/libavcodec/avcodec.h:5386:5: note: declared here
[  444s]  int avcodec_encode_video2(AVCodecContext *avctx, AVPacket *avpkt,
[  444s]      ^~~~~~~~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:273:75: warning: 'int avcodec_encode_video2(AVCodecContext*, AVPacket*, const AVFrame*, int*)' is deprecated [-Wdeprecated-declarations]
[  444s]    int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, 0, &got_packet);
[  444s]                                                                            ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:6:
[  444s] /usr/include/ffmpeg/libavcodec/avcodec.h:5386:5: note: declared here
[  444s]  int avcodec_encode_video2(AVCodecContext *avctx, AVPacket *avpkt,
[  444s]      ^~~~~~~~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:287:35: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   avcodec_close(m_ff->videoStream->codec);
[  444s]                                    ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:287:35: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   avcodec_close(m_ff->videoStream->codec);
[  444s]                                    ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:287:35: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]   avcodec_close(m_ff->videoStream->codec);
[  444s]                                    ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:293:46: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]    av_freep(&m_ff->formatContext->streams[i]->codec);
[  444s]                                               ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:293:46: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]    av_freep(&m_ff->formatContext->streams[i]->codec);
[  444s]                                               ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:293:46: warning: 'AVStream::codec' is deprecated [-Wdeprecated-declarations]
[  444s]    av_freep(&m_ff->formatContext->streams[i]->codec);
[  444s]                                               ^~~~~
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:7:
[  444s] /usr/include/ffmpeg/libavformat/avformat.h:877:21: note: declared here
[  444s]      AVCodecContext *codec;
[  444s]                      ^~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp: In member function 'virtual bool QVideoEncoder::encodeImage(const QImage&, int, QString*)':
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:334:85: warning: 'int avcodec_encode_video2(AVCodecContext*, AVPacket*, const AVFrame*, int*)' is deprecated [-Wdeprecated-declarations]
[  444s]    int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, m_ff->frame, &got_packet);
[  444s]                                                                                      ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:6:
[  444s] /usr/include/ffmpeg/libavcodec/avcodec.h:5386:5: note: declared here
[  444s]  int avcodec_encode_video2(AVCodecContext *avctx, AVPacket *avpkt,
[  444s]      ^~~~~~~~~~~~~~~~~~~~~
[  444s] /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:334:85: warning: 'int avcodec_encode_video2(AVCodecContext*, AVPacket*, const AVFrame*, int*)' is deprecated [-Wdeprecated-declarations]
[  444s]    int ret = avcodec_encode_video2(m_ff->codecContext, &pkt, m_ff->frame, &got_packet);
[  444s]                                                                                      ^
[  444s] In file included from /home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/plugins/qAnimation/QTFFmpegWrapper/QVideoEncoder.cpp:6:
[  444s] /usr/include/ffmpeg/libavcodec/avcodec.h:5386:5: note: declared here
[  444s]  int avcodec_encode_video2(AVCodecContext *avctx, AVPacket *avpkt,
[  444s]      ^~~~~~~~~~~~~~~~~~~~~
[  444s] make[2]: *** [plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/build.make:66: plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/QVideoEncoder.cpp.o] Error 1
[  444s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  444s] make[1]: *** [CMakeFiles/Makefile2:1798: plugins/qAnimation/QTFFmpegWrapper/CMakeFiles/QTFFMPEG_LIB.dir/all] Error 2
[  444s] make[1]: Leaving directory '/home/abuild/rpmbuild/BUILD/CloudCompare-2.9.1/build'
[  444s] make: *** [Makefile:133: all] Error 2
[  444s] error: Bad exit status from /var/tmp/rpm-tmp.QqJ2GR (%build)
```

**Comment by dgirardeau** [comment]
Apparently the only 'real' error is about the CODEC_FLAG_GLOBAL_HEADER definition in QVideoEncoder.cpp (line 182). The compiler even suggests to use 'AV_CODEC_FLAG_GLOBAL_HEADER'. Have you tried that?

**Comment by asmaloney** [comment]
FYI this was fixed in the current master branch in April: 8488a227978c7765fd05c99966d858725b510bbc

**Comment by dgirardeau** [comment]
Ah :D
